### ISSUE TITLE
Do sponsorships in kind (e.g. hardware) or service (e.g. hosting, monitoring, etc) count?

### ISSUE THREAD

**Issue by jankeromnes** [ossfriendly/open-source-supporters#6]

**Comment by kof** [comment]
Is it Open Source hardware? Does it reduce the overall debt?

**Comment by tbarn** [comment]
I see the update from #22, but it didn't really answer the question about in-kind. 

Any thoughts on this? @staltz

**Comment by staltz** [comment]
Just my opinion (which is not enough to define this list maintained not just by me):

Sponsorship in kind and service are cool, and certainly help. However the problem we're looking to solve is this: 

> Open source maintainers are often an invisible and underfunded workforce behind our informational infrastructure. With this list we would like to encourage more companies to be aware of that and play active role in supporting our common grounds.

I wouldn't consider giving coupons to *any* underfunded workforce an adequate support and solution to pay them back. I like analogies, and the analogy I see here is giving a shiny new axe for a miner to use at work, while in reality the miner is underpaid. So my answer would be no, despite shiny axes for free being really cool, and still useful.

**Comment by ShaneCurcuru** [comment]
There's a simple answer to most of the questions that are going to come up as you design this: "It depends."  The answer of "is this contribution some company wants to give us valuable?" depends on every specific project or foundation.  Sometimes, it's purely a marketing game by the donating company; good for their press release, but it doesn't actually help the FOSS projects.  Sometimes it's an important but hidden value.

The ASF has a wide variety of financial and in-kind sponsors.  In-kind sponsors of the foundation are listed separately under Infrastructure sponsors:
https://www.apache.org/foundation/thanks.html

To get on this page, organizations need to provide a notable amount of value to the ASF and our projects for bandwidth, hosting, graphic design, and other non-code-development work *that the ASF needs*.  

Separately, many companies donate software licenses to some or all Apache projects.  Those are recognized on individual project pages.  That's where it gets tricky - plenty of companies donate (sometimes expensive) licenses to development software, but how important is that software to the various volunteers working on the project?  I've seen both kinds of donations.  Some projects have lots of committers from different backgrounds who can really use the donated software for their project work - either as part of their dayjob, or just as a hobbyist.  But some of the donated licenses are not that important to the project.  

TL;DR: it's complicated, so spending a little time quantifying how you'll track different kinds of support is a good idea.  Even better would be to prototype the classifications with a couple of different FOSS projects - preferably ones with an organized governance, to make it more accurate - before making a wider call for maintainers to submit data.

**Comment by kof** [comment]
Yeah, thats a good summary for why small open source projects are left out and why we need to get information about received support from the projects, not from the companies. Maintainers should decide what they want to receive and make it transparent.

**Comment by staltz** [comment]
👍 to Shane's analysis. I can certainly see how donations in kind can be beneficial depending on the case. And I assume donations in money and time are always universally useful because even if a company gives 10$/month, once many companies or individuals do this, it's a significant amount. The same can't be said for a large amount of licenses.
